docs/docs/contents/reference/Saturn.AzureFunctions/Saturn.html

```diff
@@ -0,0 +1,252 @@
+<html class="js csstransforms3d">
```

Comment:
In the original implementation, I believe that this file was automatically build based on some F# files and comments. For now, I don't want to replicate this feature since I think it would take a long time. We can keep updating it manually in the meantime.